app/controllers/api/auth/episodes_controller.rb

```diff
@@ -7,13 +7,26 @@ class Api::Auth::EpisodesController < Api::BaseController
   filter_resources_by :podcast_id
   find_method :find_by_guid
 
+  after_action :process_media, only: [:create, :update]
```

Comment:
Here it is, line 15 below in `show`, the super call. That doesn't work if it falls through to the episodes_controller, b/c it does different checks on the request - I think I can do something about that though.